plugin.json

```
@@ -4,31 +4,36 @@
     "description": "Microsoft Calendar Integration",
     "homepage_url": "https://mattermost.gitbook.io/plugin-mscalendar",
     "support_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/issues",
-    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/releases/tag/v1.1.0",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/releases/tag/v1.2.0",
     "icon_path": "assets/profile.svg",
-    "version": "1.1.0",
+    "version": "1.2.0",
     "min_server_version": "5.37.0",
     "server": {
         "executables": {
-            "linux-amd64": "server/dist/plugin-linux-amd64",
             "darwin-amd64": "server/dist/plugin-darwin-amd64",
+            "linux-amd64": "server/dist/plugin-linux-amd64",
             "windows-amd64": "server/dist/plugin-windows-amd64.exe"
-        }
+        },
+        "executable": ""
     },
     "settings_schema": {
         "header": "",
+        "footer": "",
         "settings": [
             {
                 "key": "AdminUserIDs",
                 "display_name": "Admin User IDs:",
                 "type": "text",
-                "help_text": "List of users authorized to administer the plugin in addition to the System Admins. Must be a comma-separated list of user IDs.\n \n User IDs can be found in **System Console > User Management > Users**. Select the user's name, and the ID is displayed in the top-right corner of the banner."
+                "help_text": "List of users authorized to administer the plugin in addition to the System Admins. Must be a comma-separated list of user IDs.\n \n User IDs can be found in **System Console \u003e User Management \u003e Users**. Select the user's name, and the ID is displayed in the top-right corner of the banner.",
```

I wonder why those right carets were changed

```suggestion
                "help_text": "List of users authorized to administer the plugin in addition to the System Admins. Must be a comma-separated list of user IDs.\n \n User IDs can be found in **System Console > User Management > Users**. Select the user's name, and the ID is displayed in the top-right corner of the banner.",
```